crates/cxx-qt-gen/test_inputs/shebang.rs

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/
+// Test for qtfile
```

Comment:
i would still have one item in here and then in your `test_parse_qt_file_shebang_strip` have those checks that are in the other one

```
        assert!(qt_file.attrs.is_empty());
        assert_eq!(qt_file.items.len(), 1);
        assert!(matches!(qt_file.items.first(), Some(CxxQt(_))))
````

Just so that you ensure that when there is a shebang, yes we strip it but do we also still parse the remainder of the file OK?